treon/treon.py

```diff
@@ -46,6 +51,18 @@ def main():
         sys.exit(-1)
 
 
+def setup_logging(arguments):
+    logging.basicConfig(level=loglevel(arguments), format='%(message)s')
+
+    # Silence internal logging from notebook execution
+    logging.getLogger('traitlets').setLevel(logging.CRITICAL)
```

Comment:
ah, good find!

Reply:
Ah, like two hours after opening the PR I suddenly remembered how things were supposed to work again 😕

Is: I was configuring the root logger, and by accident all third-party loggers, which inherit the log level. I chose to silence them using above kludge.

Should be: a dedicated logger for treon, whose level we set, leaving the rest alone.

Can be: we may chose to obtain full debug output, including logging internal to Jupyter, at a later stage.

Reply:
I see. For now should we enable logging from notebook for verbose mode? It's as simple as not applying this silencer in case of verbose mode. 

Something like that would be useful for users who are debugging test failures. What do you think?

Reply:
Currently, I do not think the internal Jupyter logging is relevant. Just now it's really noisy. For example, the notebook contents get printed again. This is why I did not pursue it.

Reply:
I see. I thought Jupyter logging meant folks logging stuff in their notebook cells. This looks more like logging coming from jupyter/nbconvert codebase. 

Anyway, we can keep it as is and address it if somebody faces a problem.

Reply:
Yes, exactly. Perhaps I should have explained a little bit more upfront.